CHANGELOG.md

```diff
@@ -25,7 +25,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   control if iDynTree is built as a shared or a static library.
 - The Python method `*.fromPyList` is replaced by `*.FromPython` (https://github.com/robotology/idyntree/pull/726).
 - The minimum required CMake version to configure and compile iDynTree is now 3.16 (https://github.com/robotology/idyntree/pull/732).
-- Import the Python SWIG bindings changed from `import iDynTree` to either `from idyntree import iDynTree` or `import idyntree.iDynTree` (https://github.com/robotology/idyntree/pull/733).
+- Import the Python SWIG bindings changed from `import iDynTree` to either `from idyntree import bindings` or `import idyntree.bindings` (https://github.com/robotology/idyntree/pull/733, https://github.com/robotology/idyntree/pull/735).
```

Comment:
I am probably a bit ignorant myself, but I am afraid this ignorance may be shared with our users, but I would go for something more verbose, such as:
> The namespace of the Python SWIG bindings changed from `iDynTree` to either `idyntree.bindings` (https://github.com/robotology/idyntree/pull/733, https://github.com/robotology/idyntree/pull/735). To continues to refer to iDynTree classes as `iDynTree.<ClassName>`, you can change your `import iDynTree` statements to `import idyntree.bindings as iDynTree`. Otherwise, you can use `from idyntree import bindings` to refer them as `bindings.<ClassName>` or `import idyntree.bindings` to refer them as `idyntree.bindings.<ClassName>` .

Note that I am not sure about the used Python jargon and if the indicated usage is correct.

Reply:
Probably we can avoid mentioning `from idyntree import bindings` ?

Reply:
I agree on both suggestions, done in https://github.com/robotology/idyntree/compare/e15bbb8fdf88a44ba83acdfa0817a9843db9bedd..e141d6bfd297a5dd13ae739a1f08ad76a12357c1.